extensions/amp-auto-ads/0.1/ad-network-config.js

```diff
@@ -65,16 +67,16 @@ class AdSenseNetworkConfig {
 
   /** @override */
   getConfigUrl() {
+    const docInfo = documentInfoForDoc(this.autoAmpAdsElement_);
+    const canonicalHostname = parseUrl(docInfo.canonicalUrl).hostname;
```

Comment:
In what way is it not reliable?

We need to use the same URL in the config request as is used as the page URL in the AdSense ad requests, which seems to be the canonical URL: https://github.com/ampproject/amphtml/blob/a8acbd2dac5f8d2835c934e3f7adccd62902941a/ads/google/adsense.js#L56.